inttest/common/footloosesuite.go

```diff
@@ -111,6 +116,18 @@ func (s *FootlooseSuite) initializeDefaults() {
 	if s.KubeAPIExternalPort == 0 {
 		s.KubeAPIExternalPort = 6443
 	}
+	if s.LaunchMode == "" {
+		s.LaunchMode = LaunchModeStandalone
+	}
+
+	switch s.LaunchMode {
+	case LaunchModeStandalone:
+		s.launchDelegate = s.StandaloneLaunchDelegate()
+	case LaunchModeOpenRC:
+		s.launchDelegate = s.OpenRCLaunchDelegate()
+	default:
+		panic("Missing launch delegate (standalone, openrc)")
```

Comment:
Would it be possible to use `s.Require().Fail(...)` instead of panicking?

Reply:
Yes, a much better idea. Done.